be/src/runtime/memory/mem_tracker_limiter.cpp

```diff
@@ -113,8 +113,17 @@ MemTracker::Snapshot MemTrackerLimiter::make_snapshot() const {
 
 void MemTrackerLimiter::refresh_global_counter() {
     std::unordered_map<Type, int64_t> type_mem_sum = {
-            {Type::GLOBAL, 0},        {Type::QUERY, 0}, {Type::LOAD, 0}, {Type::COMPACTION, 0},
-            {Type::SCHEMA_CHANGE, 0}, {Type::CLONE, 0}}; // No need refresh Type::EXPERIMENTAL
+            {Type::GLOBAL, 0},
+            {Type::QUERY, 0},
+            {Type::LOAD, 0},
+            {Type::SCHEMA_CHANGE, 0},
+            {Type::CLONE, 0},
+            {Type::OTHER_COMPACTION, 0},
+            {Type::BASE_COMPACTION, 0},
+            {Type::CUMULATIVE_COMPACTION, 0},
+            {Type::ROWID_CONVERSION, 0},
```

Comment:
It's quite wired to mix the memtracker of structures like `PrimaryKeyIndex`, `BloomFilter`, `RowIdConversion` with operations like `Load`, `Clone`, `Query` etc...